Framework/PythonInterface/plugins/algorithms/LoadPreNexusLive.py

```diff
@@ -60,6 +72,9 @@ def PyInit(self):
                              StringListValidator(instruments),
                              'Empty uses default instrument')
 
+        self.declareProperty('RunNumber', 0,
```

Comment:
Wouldn't a better default value be `Property.EMPTY_INT`?

Then do `if runNumber != Property.EMPTY_INT:` above.